docs/modules/reference/pages/service-assurance/monitors/StrafePingMonitor.adoc

```diff
@@ -103,19 +103,27 @@ Note that you must include the `monitor` section for each service in your defini
     <rra>RRA:MIN:0.5:288:366</rra>
   </rrd>
   <service name="StrafePing" interval="300000" user-defined="false" status="on">
-    <parameter key="retry" value="0"/>
-    <parameter key="timeout" value="3000"/>
-    <parameter key="ping-count" value="20"/>
-    <parameter key="failure-ping-count" value="20"/>
-    <parameter key="wait-interval" value="50"/>
-    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/>
-    <parameter key="rrd-base-name" value="strafeping"/>
+    <parameter key="retry" value="0"/> <2>
+    <parameter key="timeout" value="3000"/> <3>
+    <parameter key="ping-count" value="20"/> <4>
+    <parameter key="failure-ping-count" value="20"/> <5>
+    <parameter key="wait-interval" value="50"/> <6>
+    <parameter key="rrd-repository" value="/opt/opennms/share/rrd/response"/> <7>
+    <parameter key="rrd-base-name" value="strafeping"/> <8>
   </service>
   <downtime interval="30000" begin="0" end="300000"/>
   <downtime interval="300000" begin="300000" end="43200000"/>
   <downtime interval="600000" begin="43200000" end="432000000"/>
   <downtime begin="432000000" delete="true"/>
 </package>
-<monitor service="StrafePing" class-name="org.opennms.netmgt.poller.monitors.StrafePingMonitor"/> <1>
+<monitor service="StrafePing" class-name="org.opennms.netmgt.poller.monitors.StrafePingMonitor"/> <9>
 ----
-<1> Required *monitor* section.
\ No newline at end of file
+<1> Range of IP addresses to include - begin/end
+<2> Number of attempts to test a service’s status
+<3> Timeout for the isReachable method, in milliseconds.
```

Comment:
```suggestion
<3> Timeout for the `isReachable` method, in milliseconds.
```

Reply:
Markdown isn't happy with the adoc format, the markdown gets semi lost when rendered [no gray background]. This will need to be resolved likely a different way.